flavors/swagger-ui-react/release/run.sh

```diff
@@ -11,8 +11,8 @@ mkdir -p ../dist
 # Copy UI's dist files to our directory
 cp ../../../dist/swagger-ui.js ../dist
 cp ../../../dist/swagger-ui.js.map ../dist
-cp ../../../dist/swagger-ui-es-bundle.js ../dist
-cp ../../../dist/swagger-ui-es-bundle.js.map ../dist
+cp ../../../dist/swagger-ui-es-bundle-core.js ../dist
+cp ../../../dist/swagger-ui-es-bundle-core.js.map ../dist
 cp ../../../dist/swagger-ui.css ../dist
 cp ../../../dist/swagger-ui.css.map ../dist
```

Comment:
In order to fix the `swagger-ui-react` problem we have to make following changes:

### Change which files are copied

From:
```sh
cp ../../../dist/swagger-ui.js ../dist
cp ../../../dist/swagger-ui.js.map ../dist
cp ../../../dist/swagger-ui-es-bundle-core.js ../dist
cp ../../../dist/swagger-ui-es-bundle-core.js.map ../dist
cp ../../../dist/swagger-ui.css ../dist
cp ../../../dist/swagger-ui.css.map ../dist
```
To
```sh
cp ../../../dist/swagger-ui-es-bundle-core.js ../dist
cp ../../../dist/swagger-ui-es-bundle-core.js.map ../dist
cp ../../../dist/swagger-ui.css ../dist
cp ../../../dist/swagger-ui.css.map ../dist
```

Explanation: we don't use `swagger-ui.js` UMD build fragment anymore here.

### Change the import in `index.js` file

From:
```js
import swaggerUIConstructor, {presets} from "./swagger-ui"
```
To
```js
import swaggerUIConstructor, {presets} from "./swagger-ui-es-bundle-core"
```

Explanation: we want to use the `commonjs2` build fragment here so the import needs to reflect that.

--- 
I've tested on CRA and these changes makes `swagger-ui-react` work again.

Reply:
Can you share your CRA example? Still not working for me. In addition, I tried `swagger-ui-react` in a different project, please see separate issue [comment](https://github.com/swagger-api/swagger-ui/pull/7826#issuecomment-1043613869)